pdb2pqr/debump.py

```diff
@@ -126,7 +126,9 @@ def get_bump_score_atom(self, atom):
             )
             cutoff = atom_size + other_size
             if dist < cutoff:
-                bumpscore = bumpscore + 1000.0
+                bumpscore += 1000.0
+                # Also ignore if this is a donor/acceptor pair
+                pair_ignored = False
```

Comment:
Doesn't this mean the pair is always ignored?

Reply:
Sorry, I should have put a warning and a question for @sobolevnrm there. The `pair_ignored` is never initialized in `debump.py` so there has been a logic error in the `if pair_ignored` for some time. I don't know the correct logic to implement to determine if a pair gets ignored or not. The current implementation just always logs a debug message.

Reply:
Should we create an issue for this one?